tests/evaluation/test_yarrow.py

```diff
@@ -0,0 +1,57 @@
+import logging
```

Comment:
we should remove yarrow and make it test whitenoise_core

Reply:
Updated yarrow to whitenoise-core in all the files.